js/main_nav.js

```diff
@@ -6,6 +6,11 @@ let navBurguer = document.getElementById("main-nav__burguer");
 
 let overlayMenu = document.getElementById("main-nav__menu-items");
 
+let logoElement = document.querySelector(".homepage-hero__logo");
+let logoHeight = logoElement.offsetHeight;
+let logoTop = logoElement.offsetTop;
```

Comment:
I believe this variable might be named a little more explanatory, like logoOffsetTop, what do you think? Something that help us understand the variable faster. It's a thought.